app/code/Magento/CatalogImportExport/Model/Import/Product.php

```diff
@@ -2406,6 +2412,17 @@ public function validateRow(array $rowData, $rowNum)
      */
     private function isNeedToValidateUrlKey($rowData)
     {
+        /**
+         * If the product exists, assume it already has a URL Key and even
+         * if a name is provided in the import data, it should not be used
+         * to overwrite that existing URL Key the product already has.
+         * @see https://github.com/magento/magento2/issues/17023
```

Comment:
Comments are good, can you please remove Magento issues links from the code. Let keep it clean.